README.md

```diff
@@ -1,58 +1,44 @@
+  [simulation]: https://github.com/near/near-sdk-rs/tree/master/near-sdk-sim
 
-_Using Gitpod? You can skip these setup steps!_
+## Building
 
-To run this project locally:
+Follow instructions for installing Rust here https://docs.near.org/docs/tutorials/contracts/intro-to-rust#3-step-rust-installation (if you're using Gitpod, you can skip this step).
 
-1. Prerequisites: Make sure you have Node.js ≥ 12 installed (https://nodejs.org), then use it to install [yarn]: `npm install --global yarn` (or just `npm i -g yarn`)
-2. Install dependencies: `yarn install` (or just `yarn`)
-3. Follow instructions for installing [rust] here https://docs.near.org/docs/tutorials/contracts/intro-to-rust#3-step-rust-installation
+To build run:
+```bash
+./build.sh
+```
 
-Now you can run all the [rust]-related scripts listed in `package.json`! Scripts you might want to start with:
+## Testing
 
-- `yarn test:unit:rs`: Runs all Rust tests in the project
-- `yarn build:rs`: Compiles the Rust contracts to [Wasm] binaries
+As with many Rust libraries and contracts, there are tests in the main fungible token implementation at `ft/src/lib.rs`.
 
+Additionally, this project has [simulation] tests in `tests/general.rs`. Simulation tests allow testing cross-contract calls, which is crucial to ensuring that the `ft_transfer_call` function works properly. These simulation tests are the reason this project has the file structure it does. Note that the root project has a `Cargo.toml` which sets it up as a workspace. `ft` and `test-contract-defi` are both small & focused contract projects, the latter only existing for simulation tests. The root project imports `near-sdk-sim` and tests interaction between these contracts.
 
-# AssemblyScript
+You can run all these tests with one command:
 
-_Using Gitpod? You can skip these setup steps!_
+```bash
+cargo test
+```
 
-To run this project locally:
+If you want to run only simulation tests, you can use `cargo test simulation`, since all the simulation tests include "simulation" in their names.
 
-1. Prerequisites: Make sure you have Node.js ≥ 12 installed (https://nodejs.org), then use it to install [yarn]: `npm install --global yarn` (or just `npm i -g yarn`)
-2. Install dependencies: `yarn install` (or just `yarn`)
 
-Now you can run all the [AssemblyScript]-related scripts listed in `package.json`! Scripts you might want to start with:
+## Notes
 
-- `yarn test:unit:as`: Runs all AssemblyScript tests with filenames ending in
-  `unit.spec`
-- `yarn build:as`: Compiles the AssemblyScript contracts to [Wasm] binaries
+ - The maximum balance value is limited by U128 (`2**128 - 1`).
+ - JSON calls should pass U128 as a base-10 string. E.g. "100".
+ - This does not include escrow functionality, as `ft_transfer_call` provides a superior approach. An escrow system can, of course, be added as a separate contract or additional functionality within this contract.
```

Comment:
These notes were pulled from `near-sdk-rs`. I'm not sure if this is the best way to incorporate this info into this example. Thoughts?

Reply:
I think this works 👍🏼